diff --git a/node_modules/secretjs/dist/secret_network_client.js b/node_modules/secretjs/dist/secret_network_client.js
index 6379a5a..2e921fc 100644
--- a/node_modules/secretjs/dist/secret_network_client.js
+++ b/node_modules/secretjs/dist/secret_network_client.js
@@ -32,8 +32,6 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.TxResultCode = exports.gasToFee = exports.SecretNetworkClient = exports.ReadonlySigner = exports.BroadcastMode = void 0;
-const cross_fetch_1 = __importDefault(require("cross-fetch"));
-global.fetch = cross_fetch_1.default;
 const encoding_1 = require("@cosmjs/encoding");
 const sha256_1 = require("@noble/hashes/sha256");
 const _1 = require(".");
diff --git a/node_modules/secretjs/src/secret_network_client.ts b/node_modules/secretjs/src/secret_network_client.ts
index b8aafdb..b6d9d70 100644
--- a/node_modules/secretjs/src/secret_network_client.ts
+++ b/node_modules/secretjs/src/secret_network_client.ts
@@ -1,6 +1,3 @@
-import fetch from "cross-fetch";
-global.fetch = fetch;
-
 import {
   fromBase64,
   fromHex,
@@ -8,6 +5,8 @@ import {
   toBase64,
   toHex,
 } from "@cosmjs/encoding";
+
+
 import { sha256 } from "@noble/hashes/sha256";
 import {
   Coin,
